lib/src/index.ts

```diff
@@ -1,6 +1,6 @@
-export { QueryParams } from "./liveapp";
-export type { FeeStrategy } from "./sdk";
-export { ExchangeSDK } from "./sdk";
+export { QueryParams } from "./liveapp.js";
```

Comment:
Do we need to have these `.js` file extension?

Reply:
For `"type": "module"` in the package.json I think so.